src/v/cloud_storage/read_path_probes.cc

```diff
@@ -136,6 +136,13 @@ ts_read_path_probe::ts_read_path_probe() {
           },
           sm::description("Chunk hydration latency histogram"))
           .aggregate(aggregate_labels),
+
+        sm::make_counter(
+          "downloads_throttled_sum",
```

Comment:
Feels like this metric will be important/useful. Should we include it in public metrics too?

Reply:
done